tests/specs/ecosystem/bureaudouble/vaul/0_0_2.test

```diff
@@ -1,252 +1,253 @@
-      "npm:react-dom@^18.0.0": "npm:react-dom@18.3.1_react@18.3.1",
-      "npm:react@^18.2.0": "npm:react@18.3.1"
+  "version": "4",
+  "specifiers": {
+    "jsr:@bureaudouble-forks/aria-hidden@0.0.1": "jsr:@bureaudouble-forks/aria-hidden@0.0.1",
+    "jsr:@bureaudouble-forks/react-remove-scroll-bar@0.0.2": "jsr:@bureaudouble-forks/react-remove-scroll-bar@0.0.2",
+    "jsr:@bureaudouble-forks/react-remove-scroll@0.0.1": "jsr:@bureaudouble-forks/react-remove-scroll@0.0.1",
+    "jsr:@bureaudouble-forks/react-style-singleton@0.0.1": "jsr:@bureaudouble-forks/react-style-singleton@0.0.1",
+    "jsr:@bureaudouble-forks/use-callback-ref@0.0.1": "jsr:@bureaudouble-forks/use-callback-ref@0.0.1",
+    "jsr:@bureaudouble-forks/use-sidecar@0.0.1": "jsr:@bureaudouble-forks/use-sidecar@0.0.1",
+    "jsr:@radix-ui-fork/primitive@0.0.6": "jsr:@radix-ui-fork/primitive@0.0.6",
+    "jsr:@radix-ui-fork/react-compose-refs@0.0.6": "jsr:@radix-ui-fork/react-compose-refs@0.0.6",
+    "jsr:@radix-ui-fork/react-context@0.0.6": "jsr:@radix-ui-fork/react-context@0.0.6",
+    "jsr:@radix-ui-fork/react-dialog@^0.0.6": "jsr:@radix-ui-fork/react-dialog@0.0.6",
+    "jsr:@radix-ui-fork/react-dismissable-layer@0.0.6": "jsr:@radix-ui-fork/react-dismissable-layer@0.0.6",
+    "jsr:@radix-ui-fork/react-focus-guards@0.0.6": "jsr:@radix-ui-fork/react-focus-guards@0.0.6",
+    "jsr:@radix-ui-fork/react-focus-scope@0.0.6": "jsr:@radix-ui-fork/react-focus-scope@0.0.6",
+    "jsr:@radix-ui-fork/react-id@0.0.6": "jsr:@radix-ui-fork/react-id@0.0.6",
+    "jsr:@radix-ui-fork/react-portal@0.0.6": "jsr:@radix-ui-fork/react-portal@0.0.6",
+    "jsr:@radix-ui-fork/react-presence@0.0.6": "jsr:@radix-ui-fork/react-presence@0.0.6",
+    "jsr:@radix-ui-fork/react-primitive@0.0.6": "jsr:@radix-ui-fork/react-primitive@0.0.6",
+    "jsr:@radix-ui-fork/react-slot@0.0.6": "jsr:@radix-ui-fork/react-slot@0.0.6",
+    "jsr:@radix-ui-fork/react-use-callback-ref@0.0.6": "jsr:@radix-ui-fork/react-use-callback-ref@0.0.6",
+    "jsr:@radix-ui-fork/react-use-controllable-state@0.0.6": "jsr:@radix-ui-fork/react-use-controllable-state@0.0.6",
+    "jsr:@radix-ui-fork/react-use-escape-keydown@0.0.6": "jsr:@radix-ui-fork/react-use-escape-keydown@0.0.6",
+    "jsr:@radix-ui-fork/react-use-layout-effect@0.0.6": "jsr:@radix-ui-fork/react-use-layout-effect@0.0.6",
+    "npm:@types/react@^18.2.0": "npm:@types/react@18.3.1",
+    "npm:react-dom@^18.0.0": "npm:react-dom@18.3.1_react@18.3.1",
+    "npm:react@^18.2.0": "npm:react@18.3.1"
+  },
+  "jsr": {
+    "@bureaudouble-forks/aria-hidden@0.0.1": {
+      "integrity": "9f5900d6e362bc1000d1c2db284841140ab3633066a48b0949cd1c5270142908"
+    },
+    "@bureaudouble-forks/react-remove-scroll-bar@0.0.2": {
+      "integrity": "30196c385c68d1281f4656ff183b71c542d11008cdca448438c029d196ab83e3",
+      "dependencies": [
+        "jsr:@bureaudouble-forks/react-style-singleton",
+        "npm:@types/react",
+        "npm:@types/react"
```

Comment:
There's definitely a bug here. I think it's fine to just commit this in order to unblock, but I'll fix this tomorrow.

Reply:
Actually, I reverted this. Going to look into and fix this issue first.

Reply:
This bug is caused by not using deno_semver in deno_lockfile, so duplicate equivalent reqs are ending up here.